src/MainWindow.vala

```diff
@@ -40,11 +40,26 @@ public class MainWindow : Gtk.Window {
         set_position (Gtk.WindowPosition.CENTER);
 
         var header = new Gtk.HeaderBar ();
-        header.get_style_context ().add_class ("default-decoration");
-        header.get_style_context ().add_class (Gtk.STYLE_CLASS_FLAT);
```

Comment:
Could we possibly keep this line for now? I'd like to do any style changes in a separate PR